virtcontainers/factory.go

```diff
@@ -6,6 +6,6 @@
 package virtcontainers
 
 type Factory interface {
-	GetVM(hypervisorType HypervisorType, hypervisorConfig HypervisorConfig) (*VM, error)
```

Comment:
Can this whole commit be folded into the one that added factory.go in the first place? Seems this fixes something effectively 'missing' from that prior commit?